CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file
 automatically by Versionist. DO NOT EDIT THIS FILE MANUALLY!
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## v10.0.1 - 2018-08-06
+
+* Restore disabled key generation tests for not found app/device #558 [Thodoris Greasidis]
```

Comment:
This change is on master, I don't understand how it's ended up in this PR...

Reply:
That's really interesting.
Might be GH having trouble after I FF the organizations branch, since when I do `git diff origin/organizations` locally, this isn't part of the diff.

Reply:
Yep, a rebase & squash did the trick :) The diff now looks correct in GH